static/app/views/replays/detail/network/index.tsx

```diff
@@ -234,6 +277,28 @@ function NetworkList({
             projectId={projectId}
             startTimestampMs={startTimestampMs}
           />
+          {sortConfig.by === 'startTimestamp' && showJumpUpButton() ? (
+            <Button
+              onClick={handleClick}
+              aria-label={t('Jump Up')}
+              priority="primary"
+              size="xs"
+              style={{position: 'absolute', justifySelf: 'center', top: '28px'}}
+            >
+              {t('Jump Up')}
+            </Button>
+          ) : null}
+          {sortConfig.by === 'startTimestamp' && showJumpDownButton() ? (
+            <Button
+              priority="primary"
+              size="xs"
+              onClick={handleClick}
+              aria-label={t('Jump Down')}
+              style={{position: 'absolute', justifySelf: 'center', bottom: '5px'}}
+            >
+              {t('Jump Down')}
+            </Button>
```

Comment:
Can we make this one scroll to the row-after the target index? So like, `index+1` kind of thing?

I notice that when i'm scrolling up to the top the purple line is visible, so i have the context that the scrolling worked:
<img width="667" alt="SCR-20230818-mjws" src="https://github.com/getsentry/sentry/assets/187460/f7bfdbc2-9f87-4ad9-9f5c-fcf4d2e1aea2">

But the scrolling in the down direction there's no purple line visible, but the same row is shown:
<img width="659" alt="SCR-20230818-mjyh" src="https://github.com/getsentry/sentry/assets/187460/880b4cea-0c0d-43b7-83d7-caf20a98d7f9">

What i'm going for is: if i click the 'scroll down' button i want to actually scroll a little further so it looks like this, then i can tell that the scrolling worked:
<img width="661" alt="SCR-20230818-mjzo" src="https://github.com/getsentry/sentry/assets/187460/ba577e0b-7e9a-4557-abbd-fcc8e67f772e">